astropy/config/paths.py

```diff
@@ -63,7 +55,6 @@ def _find_home():
             homedir = decodepath(os.path.join(os.environ['USERPROFILE']))
         else:
             try:
-                from ..extern.six.moves import winreg as wreg
```

Comment:
That works without replacement?

Reply:
oh, sure. there may be still a bunch of these cases that the current tests suite didn't pick up. If you're interested in diving into this, you can open a PR againts this branch, or just leave a comment for every problematic one find (it's enough to comment once per type, I'll grep+sed them for the full codebase)

Reply:
I don't have write-access to astropy/astropy so I would need to do it "old-style" (PR against your branch). I rather scan through the changes and add comments wherever I find something strange.